dsl/docker-multiarch/images/openjdk.groovy

```diff
@@ -0,0 +1,44 @@
+import vars.multiarch
```

Comment:
Fixed this. Added the override 'openjdk': 'java' to vars/multiarch.groovy and tested, image now builds as 'java'